src/GitElephant/Command/BranchCommand.php

```diff
@@ -164,9 +164,9 @@ public function singleInfo($name, $all = false, $simple = false, $verbose = fals
      * @throws \RuntimeException
      * @return string the command
      */
-    public function delete($name, $force = false)
+    public function delete($name, $force = false): string
     {
-        $arg = ($force === true) ? '-D' : '-d';
+        $arg = ($force) ? '-D' : '-d';
```

Comment:
Minor notice:
not sure this have changed to better..    would preferably added type hint "bool" at least for $force arg. 
And why the braces are left?!